testsuite/integration-arquillian/tests/base/src/main/java/org/keycloak/testsuite/util/InfinispanContainer.java

```diff
@@ -50,6 +54,16 @@ public InfinispanContainer() {
         withEnv("USER", USERNAME);
         withEnv("PASS", PASSWORD);
         withNetworkMode("host");
+        
+        Path dir = Path.of(Path.of("").toAbsolutePath() + "/target/lib");
+        Path timeTaskPath;
+        try {
+            timeTaskPath  = Files.find(dir, 1, (path, attr) -> path.toString().endsWith(".jar")).findFirst().orElse(null);
+        } catch (IOException e) {
+            throw new RuntimeException(e);
+        }
+        MountableFile mountableFile = MountableFile.forHostPath(timeTaskPath, 0666);
+        withCopyFileToContainer(mountableFile, "/opt/infinispan/server/lib/infinispan-time-task.jar");
```

Comment:
Could this copy the whole testsuite-providers jar to infinispan to the same effect?

Reply:
Yes, I changed it to testsuite-providers